tests/integration/targets/rds_instance_modify/tasks/main.yml

```diff
@@ -493,6 +493,72 @@
           - db_info.instances[0].enabled_cloudwatch_logs_exports is not defined
   # Test modifying cloudwatch log exports Complete-------------------------------------------
 
+  # Test multi tenant parameter
+
+    - name: Create a mariadb instance with multi tenant parameter
```

Comment:
We know that RDS is slow, I probably would have preferred to add some units for this new feature rather than another task that creates a new db.

Reply:
@alinabuzachis I have added the unit testcases.